metadata-ingestion/src/datahub/ingestion/source/feast_image/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Use the official lightweight Python image.
+# https://hub.docker.com/_/python
```

Comment:
just out of curiosity, why not put the python3 executable line in the image using ENTRYPOINT (so you can still pass arguments)?